spec/rails_helper.rb

```diff
@@ -25,7 +25,7 @@
 require 'capybara/poltergeist'
 Capybara.javascript_driver = :poltergeist
 Capybara.app_host = 'http://localhost'
-Capybara.server_port = 8080
+Capybara.server_port = 8081
```

Comment:
It's just a "fix up tests that are behaving annoyingly" thing. It was trying to run them on the same port as we typically run unicorn for our dev server, which caused weird behaviour at times.